packages/typescript/internal/ts_project.bzl

```diff
@@ -358,6 +428,7 @@ def ts_project_macro(
         out_dir = None,
         root_dir = None,
         link_workspace_root = False,
+        js_outs = [],
```

Comment:
why is js_outs special here? can we avoid that?
if the user has more js_outs, they might also have more map_outs?

Reply:
Hm, I am pretty sure is was one test build that passes in js_outs as an attribute. I think that's why I added it here. I don't have a strong opinion on this, although it could come in handy that if a use wants to predeclare some outputs that might be in a filegroup, that they can specify them here. I suppose the other outs could be derived from this. Let me know if you want me to proceed a particular way with this.

Reply:
let's not introduce a special case unless we know the use case for it, I'd rather keep the attribute list the same for now